src/web/HTMLIngredient.mjs

```diff
@@ -42,6 +42,17 @@ class HTMLIngredient {
             i, m;
 
         switch (this.type) {
+            case "code":
```

Comment:
As discussed above, you can probably just use the `text` type instead of creating a new argument here. See line 236 in this file for its definition. It's basically identical to what you have here.